Gopkg.toml

```diff
@@ -143,7 +143,7 @@
 
 [[constraint]]
   name = "gopkg.in/juju/charm.v6"
-  revision = "5217f4ff9d6d681861601ad653e151ffb2f05b90"
+  revision = "9a4cb2e3a67cfb52a8cddeb88b32057d3310ff73"
```

Comment:
This is superceded by c390a4cc19a3efac72d6fcce80ab2984d54f1f3f
which has landed in develop already.
So this change and the Gopkg.lock one can be removed before anding the PR.